src/trans/gpu/algor/ext_acc.F90

```diff
@@ -268,7 +268,7 @@ subroutine ext_acc_create(ptrs, stream)
     num_ranges = get_common_pointers(ptrs, common_ptrs)
 
     do i = 1, num_ranges
-      call c_f_pointer(common_ptrs(i)%ptr, pp, shape=[common_ptrs(i)%sz/sizeof(pp(1))])
+      call c_f_pointer(common_ptrs(i)%ptr, pp, shape=[common_ptrs(i)%sz/storage_size(pp(1))/8])
```

Comment:
```suggestion
      call c_f_pointer(common_ptrs(i)%ptr, pp, shape=[common_ptrs(i)%sz/(storage_size(pp(1))/8)])
```
And similar edits required further where `storage_size` is in the denominator

Reply:
Very good point. Somehow I forgot that `(x/y)/z` isn't the same as `x/(y/z)`.